diff -up libunistring-1.2/config.h.in.1~ libunistring-1.2/config.h.in
--- libunistring-1.2/config.h.in.1~	2024-02-26 21:00:59.304912825 +0100
+++ libunistring-1.2/config.h.in	2024-02-26 21:04:38.059107453 +0100
@@ -2058,7 +2058,7 @@
    can be freed via 'free'; it can be used only after declaring 'free'.  */
 /* Applies to: functions.  Cannot be used on inline functions.  */
 #ifndef _GL_ATTRIBUTE_DEALLOC
-# if _GL_GNUC_PREREQ (11, 0)
+# if _GL_GNUC_PREREQ (11, 0) && !defined(__clang__)
 #  define _GL_ATTRIBUTE_DEALLOC(f, i) __attribute__ ((__malloc__ (f, i)))
 # else
 #  define _GL_ATTRIBUTE_DEALLOC(f, i)
diff -up libunistring-1.2/gnulib-m4/gnulib-common.m4.1~ libunistring-1.2/gnulib-m4/gnulib-common.m4
--- libunistring-1.2/gnulib-m4/gnulib-common.m4.1~	2024-02-26 21:00:59.304912825 +0100
+++ libunistring-1.2/gnulib-m4/gnulib-common.m4	2024-02-26 21:04:59.871327096 +0100
@@ -222,7 +222,7 @@ AC_DEFUN([gl_COMMON_BODY], [
    can be freed via 'free'; it can be used only after declaring 'free'.  */
 /* Applies to: functions.  Cannot be used on inline functions.  */
 #ifndef _GL_ATTRIBUTE_DEALLOC
-# if _GL_GNUC_PREREQ (11, 0)
+# if _GL_GNUC_PREREQ (11, 0) && !defined(__clang__)
 #  define _GL_ATTRIBUTE_DEALLOC(f, i) __attribute__ ((__malloc__ (f, i)))
 # else
 #  define _GL_ATTRIBUTE_DEALLOC(f, i)
diff -up libunistring-1.2/lib/stdlib.in.h.1~ libunistring-1.2/lib/stdlib.in.h
--- libunistring-1.2/lib/stdlib.in.h.1~	2024-02-23 12:29:29.000000000 +0100
+++ libunistring-1.2/lib/stdlib.in.h	2024-02-26 21:00:59.308912864 +0100
@@ -108,7 +108,7 @@ struct random_data
    that can be freed by passing them as the Ith argument to the
    function F.  */
 #ifndef _GL_ATTRIBUTE_DEALLOC
-# if __GNUC__ >= 11
+# if __GNUC__ >= 11 && !defined (__clang__)
 #  define _GL_ATTRIBUTE_DEALLOC(f, i) __attribute__ ((__malloc__ (f, i)))
 # else
 #  define _GL_ATTRIBUTE_DEALLOC(f, i)
diff -up libunistring-1.2/lib/string.in.h.1~ libunistring-1.2/lib/string.in.h
--- libunistring-1.2/lib/string.in.h.1~	2024-01-29 01:58:09.000000000 +0100
+++ libunistring-1.2/lib/string.in.h	2024-02-26 21:00:59.308912864 +0100
@@ -79,7 +79,7 @@
    that can be freed by passing them as the Ith argument to the
    function F.  */
 #ifndef _GL_ATTRIBUTE_DEALLOC
-# if __GNUC__ >= 11
+# if __GNUC__ >= 11 && !defined (__clang__)
 #  define _GL_ATTRIBUTE_DEALLOC(f, i) __attribute__ ((__malloc__ (f, i)))
 # else
 #  define _GL_ATTRIBUTE_DEALLOC(f, i)
diff -up libunistring-1.2/lib/unistring/cdefs.h.1~ libunistring-1.2/lib/unistring/cdefs.h
--- libunistring-1.2/lib/unistring/cdefs.h.1~	2024-02-26 21:00:59.308912864 +0100
+++ libunistring-1.2/lib/unistring/cdefs.h	2024-02-26 21:05:15.283482366 +0100
@@ -94,7 +94,7 @@
 # endif
 #endif
 
-#if _UC_GNUC_PREREQ (11, 0)
+#if _UC_GNUC_PREREQ (11, 0) && !defined(__clang__)
 # define _UC_ATTRIBUTE_DEALLOC(f, i) __attribute__ ((__malloc__ (f, i)))
 #else
 # define _UC_ATTRIBUTE_DEALLOC(f, i)
diff -up libunistring-1.2/lib/wchar.in.h.1~ libunistring-1.2/lib/wchar.in.h
--- libunistring-1.2/lib/wchar.in.h.1~	2024-01-29 01:58:11.000000000 +0100
+++ libunistring-1.2/lib/wchar.in.h	2024-02-26 21:00:59.308912864 +0100
@@ -95,7 +95,7 @@
    that can be freed by passing them as the Ith argument to the
    function F.  */
 #ifndef _GL_ATTRIBUTE_DEALLOC
-# if __GNUC__ >= 11
+# if __GNUC__ >= 11 && !defined (__clang__)
 #  define _GL_ATTRIBUTE_DEALLOC(f, i) __attribute__ ((__malloc__ (f, i)))
 # else
 #  define _GL_ATTRIBUTE_DEALLOC(f, i)
diff -up libunistring-1.2/tests/stdio.in.h.1~ libunistring-1.2/tests/stdio.in.h
--- libunistring-1.2/tests/stdio.in.h.1~	2024-01-29 01:58:12.000000000 +0100
+++ libunistring-1.2/tests/stdio.in.h	2024-02-26 21:00:59.308912864 +0100
@@ -119,7 +119,7 @@
    that can be freed by passing them as the Ith argument to the
    function F.  */
 #ifndef _GL_ATTRIBUTE_DEALLOC
-# if __GNUC__ >= 11
+# if __GNUC__ >= 11 && !defined (__clang__)
 #  define _GL_ATTRIBUTE_DEALLOC(f, i) __attribute__ ((__malloc__ (f, i)))
 # else
 #  define _GL_ATTRIBUTE_DEALLOC(f, i)
