diff -up libunistring-1.2/Makefile.am.2~ libunistring-1.2/Makefile.am
--- libunistring-1.2/Makefile.am.2~	2024-02-26 21:05:32.399654874 +0100
+++ libunistring-1.2/Makefile.am	2024-02-26 21:05:53.615868804 +0100
@@ -60,3 +60,6 @@ gen-ChangeLog:
 	    && rm -f "$(distdir)/ChangeLog"                             \
 	    && mv "$(distdir)/ChangeLog-t" "$(distdir)/ChangeLog";      \
 	fi
+
+pkgconfigdir = $(libdir)/pkgconfig
+pkgconfig_DATA = libunistring.pc
diff -up libunistring-1.2/configure.ac.2~ libunistring-1.2/configure.ac
--- libunistring-1.2/configure.ac.2~	2023-08-30 13:08:13.000000000 +0200
+++ libunistring-1.2/configure.ac	2024-02-26 21:05:32.399654874 +0100
@@ -160,4 +160,5 @@ AC_CONFIG_FILES([lib/exported.sh])
 AC_CONFIG_FILES([lib/unistring/version.h:lib/unistring/version.in.h])
 AC_CONFIG_FILES([lib/unistring/woe32dll.h:lib/unistring/woe32dll.in.h])
 AC_CONFIG_FILES([tests/Makefile])
+AC_CONFIG_FILES([libunistring.pc])
 AC_OUTPUT
diff -up libunistring-1.2/libunistring.pc.in.2~ libunistring-1.2/libunistring.pc.in
--- libunistring-1.2/libunistring.pc.in.2~	2024-02-26 21:05:32.399654874 +0100
+++ libunistring-1.2/libunistring.pc.in	2024-02-26 21:05:32.399654874 +0100
@@ -0,0 +1,11 @@
+prefix=@prefix@
+exec_prefix=@exec_prefix@
+includedir=@includedir@
+libdir=@libdir@
+
+Name: libunistring
+Description: Library to manipulate Unicode strings
+Version: @PACKAGE_VERSION@
+Cflags: -I${includedir}
+Libs: -L${libdir} -lunistring
+Libs.private: @LTLIBICONV@
